packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -229,7 +244,10 @@ export function DevtoolsView(): React.ReactElement {
 						</Tooltip>
 					</>
 				) : (
-					<Waiting />
+					<>
+						<Waiting />
+						<LandingView />
+					</>
```

Comment:
This seems ok as an initial stab at it, but I think the ideal way to accomplish this is that the Home menu option is the default and selected from the start, and the logic to wait for the supported features message is inside it. So if the timeout expires waiting for the message, we still show the landing page instead of switching it for the error/retry message above this. And that error/retry message probably becomes a banner on top.

Also interested in thoughts from the team on this.

Reply:
Agreed. The menu contents will be asynchronously populated, and I believe we display a spinner in the menu while we await that initial message - I'm not sure if we need to display a spinner in the main view too. But we _should_ display an error banner or something if our supported features check times out.

Reply:
@kashms Interested in your thoughts here as well.